CMakeLists.txt

```diff
@@ -423,6 +423,7 @@ if (GENERAL OR MINER)
 	add_subdirectory(libethash)
 	if (ETHASHCL)
 		add_subdirectory(libethash-cl)
+		add_subdirectory(khronos_icd)
```

Comment:
You need to add these 3 lines after:
```
		target_include_directories(OpenCL_ICD PUBLIC ${OpenCL_INCLUDE_DIR})
		target_include_directories(OpenCLDriverStub PRIVATE ${OpenCL_INCLUDE_DIR})
		target_include_directories(IcdLog PRIVATE ${OpenCL_INCLUDE_DIR})
```

After that you can remove all `include_directories(${OpenCL_INCLUDE_DIRS})` in all CMakeLists.txt files.